Maple2.File.Ingest/Utils/Vector3ParseUtil.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Numerics;
+
+namespace Maple2.File.Ingest.Utils;
+
+public static class Vector3ParseUtil {
```

Comment:
Before we add this helper, could you try updating the parser to use `M2dVector3` instead: https://github.com/kOchirasu/Maple2.File/blob/master/Maple2.File.Parser/Xml/Item/Slot.cs#L61-L62

You can do it in this same PR: https://github.com/kOchirasu/Maple2.File/pull/11